### DISCUSSION TITLE
Guests can't see the navigation

### DISCUSSION THREAD

**Discussion by aidenmitchell** [requarks/wiki#4200] [Error / Bug Report]
Guests can't see the navigation, even though I have public pages within folders. I believe that this is a bug,

**Comment by gallogabor** [comment]
I also experinence this behavior.

I'd like to create a public/private hybrid wiki page that our customers can use to find answers in FAQ and help to setup our e-mail service for them, etc. without login.
I'd also like to have private pages to access confidential documentation for employees only. When someone logs in, the customer navigation items must hide, and private navigation items must show up.

So, I'd like to have some navigation items only for guests (users not logged in), and I like to hide these items for logged in users.

I added navigation items with permission only to Guest group, but these items not showing in navgiaton. Of course they do not show up after login (dut this is desired).
If I change permission to "all", the items show up for guests, but show up for logged in users.

Is this a feature or a bug or I missed something in the docs?
